### PR TITLE
Bug in `map_times_through`.

### PR DESCRIPTION
**Pull request by ryzhyk**

### CODE REVIEW

File: src/trace/ord/key_batch.rs [hunk 1]
```diff
@@ -329,11 +329,13 @@ where
...
-                self.cursor.child.current_key(),
-                self.cursor.child.current_diff(),
-            );
+        while self.cursor.child.valid() {
+            if self.cursor.child.key().0.less_equal(upper) {
+                logic(
+                    self.cursor.child.current_key(),
+                    self.cursor.child.current_diff(),
+                );
+            }
```
Comment (Kixiron) [inline src/trace/ord/key_batch.rs:338]: We probably need some tests covering this

Comment (ryzhyk) [reply src/trace/ord/key_batch.rs:338]: We do now, in aggregation operator test harness.  We do need dedicated unit tests for cursors though.

Comment (Kixiron) [reply src/trace/ord/key_batch.rs:338]: Do we? I mentioned it because if the coverage warning saying the function was uncovered

Comment (ryzhyk) [reply src/trace/ord/key_batch.rs:338]: Ah, I guess that one tests `val_batch`

**Pull request merged**